modules/admin/src/main/java/org/motechproject/admin/internal/service/impl/SettingsServiceImpl.java

```diff
@@ -244,24 +246,25 @@ public void saveRawFile(MultipartFile file, String filename, long bundleId) {
         Enumeration<URL> enumeration = bundle.findEntries("", "*.properties", false);
         Map<String, Properties> allDefaultProperties = new LinkedHashMap<>();
 
-        while (enumeration.hasMoreElements()) {
-            InputStream is = null;
-            URL url = enumeration.nextElement();
-            try {
-                is = url.openStream();
-                Properties defaultBundleProperties = new Properties();
-                defaultBundleProperties.load(is);
-                if (!url.getFile().isEmpty()) {
-                    //We want to store plain filename, without unnecessary slash prefix
-                    allDefaultProperties.put(url.getFile().substring(1), defaultBundleProperties);
+        if (CollectionUtils.isNotEmpty(Collections.list(enumeration))) {
```

Comment:
Thanks @jhopen  Do we currently support editing of the raw module data via the Admin module UI? (Either old one or the new one)